src/CabanaPD_ForceModels.hpp

```diff
@@ -71,6 +72,45 @@ struct BaseTemperatureModel
     }
 };
 
+// This class stores temperature parameters needed for heat transfer, but not
+// the temperature itself (stored instead in the static temperature class
+// above).
+struct BaseDynamicTemperatureModel
+{
+    double delta;
+
+    double thermal_coeff;
+    double kappa;
+    double cp;
+    bool constant_conductivity;
+
+    BaseDynamicTemperatureModel( const double _delta, const double _kappa,
+                                 const double _cp,
+                                 const bool _constant_conductivity = true )
+    {
+        set_param( _delta, _kappa, _cp, _constant_conductivity );
+    }
+
+    void set_param( const double _delta, const double _kappa, const double _cp,
+                    const bool _constant_conductivity )
+    {
+        delta = _delta;
+        kappa = _kappa;
+        cp = _cp;
+        const double d3 = _delta * _delta * _delta;
+        thermal_coeff = 9.0 / 2.0 * _kappa / pi / d3;
```

Comment:
This should be analogous to Line 52 in CabanaPD_ForceModels_PMB.hpp. I was wondering if we should use a more clear description. There, we used "c". However using "k" may not be good.